.github/docker/Dockerfile-manylinux.template

```diff
@@ -42,18 +42,19 @@ RUN yum update -y && \
     swig \
     libcurl-devel \
     libxslt-devel \
-    libxml2-devel && \
-  if [ "{{ TYPE }}{{ ARCH }}" != "2014i686" ]; then yum install -y hdf5-devel; fi && \
-  if [ -f "/etc/yum.repos.d/pgdg-91.repo" ]; then yum install -y postgresql91-devel; fi && \
+    libxml2-devel
+RUN yum install -y rust cargo || true
+RUN if [ "{{ TYPE }}{{ ARCH }}" != "2014i686" ]; then yum install -y hdf5-devel; fi
+RUN if [ -f "/etc/yum.repos.d/pgdg-91.repo" ]; then yum install -y postgresql91-devel; fi
```

Comment:
I guess these are required specifically for python3.12 only, since they are added now?
If so, I think is is good to write a comment about this for each line.

Reply:
If it is not needed I think we should remove it. If the ci.workflow does not fail I think it is safe to remove it.

Reply:
done